src/lib/structures/contexts/CommandContext.ts

```diff
@@ -104,7 +104,7 @@ export class CommandContext<
 			| MessagePayload
 			| InteractionReplyOptions,
 	): Promise<Fetch extends true ? GuildCacheMessage<Cached> : void> {
-		return this.deferred || this.replied
+		return this.deferred || this.replied || this.interaction.deferred || this.interaction.replied
```

Comment:
```suggestion
		return this.deferred || this.replied || this.interaction?.deferred || this.interaction?.replied
```
Interaction can be undefined